src/random.js

```diff
@@ -0,0 +1,26 @@
+const faker = require('faker')
+const _random = require('lodash/random')
```

Comment:
Alias this as `randomInt` so it's easier to tell what kind of value it returns. There is already another `random` in here so it's best to be specific.